docker-compose.yml

```diff
@@ -105,7 +105,7 @@ services:
       test: "exit 0"
   flower:
     image: openconceptlab/oclapi:${ENVIRONMENT-production}
-    command: celery -A tasks flower --basic_auth=root:${ROOT_PASSWORD-Root123} --conf=flowerconfig.py
+    command: celery -A tasks flower --basic_auth=${FLOWER_USER-floweruser}:${FLOWER_PWD-Flower123} --conf=flowerconfig.py
```

Comment:
Could you please update https://github.com/OpenConceptLab/oclapi/blob/master/README.md#env-file-details and https://github.com/OpenConceptLab/oclapi/blob/master/docker-compose.yml#L34 to include that new variable?

Also please update docker-compose for all 4 environments. See https://github.com/openmrs/openmrs-contrib-ansible-docker-compose/blob/master/files/oclapi-prd/docker-compose.yml and corresponding ones.

Reply:
The PR has been updated.

I have also created PR for openmrs repo - https://github.com/openmrs/openmrs-contrib-ansible-docker-compose/pull/44

Reply:
Thanks!